src/DynamoCoreWpf/TestInfrastructure/CustomNodeCompatibilityMutator.cs

```diff
@@ -236,14 +236,16 @@ public List<Type> LoadAllTypesFromDynamoAssemblies()
 
```

Comment:
`DynamoModel.PathManager` now stores and manages the list of `nodes` folders internally.